### PR TITLE
#68 fix LT-20 Latitude calculation

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/globalsat/LT-20/uplink.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ function consume(event) {
   var commandID = Bits.bitsToUnsigned(bits.substr(8, 8));
 
   data.longitude = (toLittleEndian(payload.substr(4, 6)) * 215) / 10 * 0.000001;
-  data.latitude = (toLittleEndian(payload.substr(10, 6)) * 215) / 10 * 0.000001;
+  data.latitude = (toLittleEndian(payload.substr(10, 6)) * 108) / 10 * 0.000001;
```
Comment (mariusgiger) [inline types/globalsat/LT-20/uplink.js:24]: are negative numbers covered with this change?

Comment (J2thatsme) [reply types/globalsat/LT-20/uplink.js:24]: no thats something i will look into ^^

File: types/globalsat/LT-20/uplink.js [hunk 2]
```diff
@@ -1,29 +1,27 @@
...
+  var hexArray = [];
+  while (hex.length >= 2) {
+    hexArray.push(hex.substring(0, 2));
+    hex = hex.substring(2, hex.length);
+  }
+  hexArray.reverse()
+  hex = hexArray.join('');
+
+  // To signed
+  var num = Bits.bitsToSigned(Bits.hexToBits(hex));
```
Comment (mariusgiger) [inline types/globalsat/LT-20/uplink.js:12]: directly return this

**Review by mariusgiger — CHANGES_REQUESTED** [review]
please add a changelog entry

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**